doc: Add known issue for Advertising Providers (#12264)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

  **Workaround:** Avoid using LTE Power Saving Mode (PSM) and extended DRX intervals longer than approximately 30 seconds. This will reduce the risk of the issue occurring, at the cost of increased power consumption.

Bluetooth libraries and services
================================

.. rst-class:: v2-4-2 v2-4-1 v2-4-0

> NCSDK-23315: The :ref:`bt_le_adv_prov_readme` has incorrect range and default value for the :kconfig:option:`CONFIG_BT_ADV_PROV_FAST_PAIR_ADV_BUF_SIZE` Kconfig option.

Reviewer: ```suggestion
NCSDK-23315: The :ref:`bt_le_adv_prov_readme` has an incorrect range and default value for the :kconfig:option:`CONFIG_BT_ADV_PROV_FAST_PAIR_ADV_BUF_SIZE` Kconfig option.
```

>   The buffer size has not been aligned to new Fast Pair not discoverable advertising data size after size of salt included in the data was increased from 1 byte to 2 bytes.

Reviewer: ```suggestion
  The buffer size has not been aligned to the new Fast Pair not discoverable advertising data size after size of the salt included in the data was increased from 1 byte to 2 bytes.
```

Other libraries
===============

.. rst-class:: v2-4-2 v2-4-1 v2-4-0

> NCSDK-23315: The :ref:`bt_le_adv_prov_readme` has improper range and default value of the :kconfig:option:`CONFIG_BT_ADV_PROV_FAST_PAIR_ADV_BUF_SIZE` Kconfig option.

Reviewer: ```suggestion
NCSDK-23315: The :ref:`bt_le_adv_prov_readme` has incorrect range and default value for the :kconfig:option:`CONFIG_BT_ADV_PROV_FAST_PAIR_ADV_BUF_SIZE` Kconfig option.
```

  The buffer size has not been aligned to new Fast Pair not discoverable advertising data size after size of salt included in the data was increased from 1 byte to 2 bytes.

>   As a consequence of it, the :ref:`peripheral_fast_pair` sample reboots on assert when it has five Account Keys and starts advertising in the Fast Pair not discoverable advertising mode.

Reviewer: ```suggestion
  Hence, the :ref:`peripheral_fast_pair` sample reboots on assert when it has five Account Keys and starts advertising in the Fast Pair not discoverable advertising mode.
```

  This happens because the buffer is too small to fit the Fast Pair not discoverable advertising data when five Account Keys are present.

  **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``a8b668e82837295962348e9e681125c2ae11bb4e``).

> 

Reviewer: We need to have a new section "Bluetooth libraries and services" and this KI could be moved to that section as this library don't belong to "other libraries".
